pkg/apis/core/types.go

```diff
@@ -3653,18 +3696,14 @@ type NodeSpec struct {
 	// The DynamicKubeletConfig feature gate must be enabled for the Kubelet to use this field
 	// +optional
 	ConfigSource *NodeConfigSource
-
-	// Deprecated. Not all kubelets will set this field. Remove field after 1.13.
-	// see: https://issues.k8s.io/61966
-	// +optional
-	DoNotUse_ExternalID string
```

Comment:
separate out this change into its own PR, this isn't just a linting fix and affects round-trippability. We may not be able to drop this in v1 Nodes

Reply:
I have reverted this commit.

Reply:
As discussed on Slack, I have renamed this var and removed `_`.